library/src/main/java/com/etrusted/android/trustbadge/library/ui/badge/TrustbadgeContext.kt

```diff
@@ -25,9 +25,14 @@
 
 package com.etrusted.android.trustbadge.library.ui.badge
 
-sealed class TrustbadgeContext(val isExpandable: Boolean = false, val sku: String = "") {
+sealed class TrustbadgeContext(
+    val isExpandable: Boolean = false,
+    val sku: String = "",
+    val trustcardState: TrustcardStateValue = TrustcardStateValue.DEFAULT
+) {
```

Comment:
What is the purpose of `TrustcardStateValue.DEFAULT` state?
Does it tell that the Trustbadge has to show as horizontally expandable/collapsible widget (like shop grade widget) rather then showing as a trust card?